### PR TITLE
Drag and drop defender

### PR DESCRIPTION
**Pull request by dgamma3**
Can drag a defender (dark blue) on a tilemap  (light green)

-had to upgrade to the latest leap library, which supported status.
 -removed test folder,as we dont need this

https://github.com/GameJam-NatureVsNeglect/super_dash/assets/1439000/369d546d-c02b-4c20-8e59-281a92ec5ef0

### CODE REVIEW

File: lib/game/entities/player.dart [hunk 1]
```diff
@@ -176,7 +176,7 @@ class Player extends JumperCharacter<SuperDashGame> {
       return respawn();
     }
 
-    final collisions = collisionInfo.otherCollisions ?? const [];
```
Comment (dgamma3) [inline lib/game/entities/player.dart:179]: called allCollsions, in the latest version of leap

File: lib/game/entities/enemy.dart [hunk 2]
```diff
@@ -82,7 +81,6 @@ class Enemy extends PhysicalEntity<SuperDashGame> {
         SpriteAnimationComponent(
           size: Vector2.all(gameRef.tileSize),
           animation: bettleAnimation,
-          anchor: Anchor.center,
```
Comment (dgamma3) [inline lib/game/entities/enemy.dart:85]: need to get rid of the anchor center, as latest leap needs it to be TopLeft (not sure why).

this is defaulted to TopLeft, if you remove the anchor

File: lib/game/status/IgnoreGravity.dart [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import 'package:leap/leap.dart';
+
+class OnIgnoreGravity extends StatusComponent with IgnoresGravity  {}
```
Comment (dgamma3) [inline lib/game/status/IgnoreGravity.dart]: StatusComponent is the reason why I upgraded to the latest leap version

**Comment by ahmetakil** [comment]
Can you do a `dart format . ` so the code is formatted to standard rules

File: lib/game/entities/Tile.dart [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+
+import 'package:flutter/material.dart';
+import 'package:leap/leap.dart';
+
+import '../status/IgnoreGravity.dart';
```
Comment (ahmetakil) [inline lib/game/entities/Tile.dart]: Let's try to use more absolute import paths instead of relative ones that is currently the more recommended approach

**Comment by ahmetakil** [comment]
LGTM

**Pull request merged**